frontend/uddhar-frontend/src/public/components/StatNumber.jsx

```
@@ -0,0 +1,31 @@
+        onUpdate(value) {
+          if (!ref.current) return;
+  
+          ref.current.textContent = value.toFixed();
+        },
+      });
+    }, [number, isInView]);
+  
+    return (
+      <div className="flex w-72 flex-col items-center py-8 sm:py-0">
+        <p className="mb-2 text-center text-7xl font-semibold sm:text-6xl">
+          <span ref={ref}></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)